src/app/portlets/content-types/fields/content-type-fields-properties-form/field-properties/relationships-property/edit-relationship.component.ts/edit-relationships.component.ts

```diff
@@ -0,0 +1,113 @@
+import { Component, OnInit, EventEmitter, Output } from '@angular/core';
+import { DotMessageService } from '@services/dot-messages-service';
+import { PaginatorService } from '@services/paginator';
+import { DotEditContentTypeCacheService } from '@portlets/content-types/services/edit-content-type-cache.service';
+import { of as observableOf, Observable } from 'rxjs';
+import { map, flatMap, toArray } from 'rxjs/operators';
+import { RelationshipService } from '@portlets/content-types/fields/content-type-fields-properties-form/field-properties/relationships-property/services/relationship.service';
+import { DotRelationship } from '@portlets/content-types/fields/shared/dot-relationship.model';
+import { DotRelationshipCardinality } from '@portlets/content-types/fields/shared/dot-relationship-cardinality.model';
+
+interface CardinalitySorted {
+    [id: number]: DotRelationshipCardinality;
+}
+@Component({
+    providers: [PaginatorService],
+    selector: 'dot-edit-relationships',
+    templateUrl: './edit-relationships.component.html'
+})
+export class EditRelationshipsComponent implements OnInit {
```

Comment:
Again the prefix in the class and file name, I know we forget about in content types but we need to keep that convention.